Esterni/PianoProgetto/components/Introduzione.tex

```diff
@@ -0,0 +1,39 @@
+\section{Introduzione}
+\subsection{Scopo del documento}
+Lo scopo del documento è quello di specificare la pianificazione che \Gruppo  adotterà per sviluppare il progetto \NomeProgetto. All'interno sono presenti le strategie per la gestione dei rischi, le suddivisioni dei compiti con il relativo preventivo dell'utilizzo delle risorse, e il modello di sviluppo secondo il quale il team ha intenzione di lavorare.
+In particolare il documento tratta i seguenti argomenti:
+\begin{itemize}
+\item analisi dei rischi;
+\item descrizione del modello di sviluppo;
+\item pianificazione delle attività e divisione dei ruoli;
+\item stima dei costi e delle risorse necessarie allo sviluppo del progetto.
+\end{itemize}
+\subsection{Scopo del prodotto}
+L'obbiettivo del progetto è quello di sviluppare un algoritmo in grado di garantire la sincronizzazione e il salvataggio di file tra server e client, appoggiandosi sul servizio \textit{Zextras Drive}.
+L'interfaccia deve essere multi piattaforma, per poter essere utilizzato nei più importanti sistemi operativi desktop esistenti.
+Inoltre il prodotto deve essere sviluppato aderendo al pattern MVC, così da permettere rapidi cambiamenti della Business logic o dell'interfaccia utenti senza eccessivo sviluppo aggiunto.
+\subsection{Glossario}
+Al fine di rendere il documento più chiaro e leggibile, si fornisce il Glossario. In questo documento saranno contenuti e descritti tutti i termini che potrebbero risultare ambigui o con un significato particolare. I termini saranno contrassegnati da una 'G' a pedice.
+\subsection{Riferimenti}
+\subsubsection{Normativi}
+\begin{itemize}
+\item \textbf{Norme Di Progetto:} Norme di Progetto;
+\item \textbf{Regolamento Organigramma e specifica tecnico-economica:} \url{https://www.math.unipd.it/~tullio/IS-1/2020/Progetto/RO.html}.
+\end{itemize}
+
+\subsubsection{Informativi}
+\begin{itemize}
+\item \textbf{Capitolato d'appalto:} \url{https://www.math.unipd.it/~tullio/IS-1/2020/Progetto/C7.pdf};
+\item\textbf{Software Engineering - Ian Sommerville - 9th Edition, 2010;}
+\item \textbf{Slide L05 del corso Ingegneria del Software - Ciclo di vita del software:} \url{https://www.math.unipd.it/~tullio/IS-1/2020/Dispense/L05.pdf};
+\item \textbf{Slide L06 del corso Ingegneria del Software - Gestione di Progetto:} \url{https://www.math.unipd.it/~tullio/IS-1/2020/Dispense/L06.pdf}.
+\end{itemize}
+
+\subsection{Scadenze}
+Il gruppo \Gruppo si impegna a rispettare le seguenti scadenze per lo sviluppo del progetto \NomeProgetto:
```

Comment:
Queste scadenze vengono poi espresse anche in seguito nel documento, per evitare ripetizioni è da decidere se tenere queste o quelle che vengono scritte in seguito